agent/cloudwatch_test.go

```diff
@@ -0,0 +1,86 @@
+package main
```

Comment:
Not sure the reason behind `cloudwatch_test.go` I would still go ahead and name it `main_test.go` and if we ever extract the cloudwatch logic to a different file we can go ahead and extract the tests as well. WDYT?

Reply:
feel free to rebase :) 👍